Makefile

```diff
@@ -306,7 +313,7 @@ init-conda:
 
 #<<<
 .PHONY: init-docker
-init-docker: docker-volumes docker-secrets
+init-docker: docker-volumes docker-secrets hosts-file
```

Comment:
We would not need to modify hosts file for non development environments (i.e. prod deployments). Therefore it would be preferable to have this as an optional step instead of a default action with init-docker